packages/@tailwindcss-upgrade/src/index.ts

```diff
@@ -163,12 +166,13 @@ async function run() {
         await Promise.allSettled(
           files.map((file) => migrateTemplate(config.designSystem, config.userConfig, file)),
         )
-      }
 
-      success('Template migration complete.')
+        success('↳ Template migration complete.')
```

Comment:
This is also moved into the loop so you can see each migration complete. I also included `↳` which looked a bit nicer, but this can be changed of course.

Reply:
@RobinMalfait I like the `↳`